templates/resources.tera

```diff
@@ -0,0 +1,26 @@
+---
+whiskers:
+  version: 2.4.0
+  matrix:
+    - flavor
+  filename: "src/catppuccin-{{ flavor.identifier }}/resources.qrc"
+---
+<!DOCTYPE RCC><RCC version="1.0">
+	<qresource>
+		<file>stylesheet.qss</file>
+		<file>config.json</file>
+
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/checkbox_checked.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/checkbox_unchecked.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/checkbox_indeterminate.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/check.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/arrow_down.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/arrow_down_disabled.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/chevron_up.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/chevron_up_disabled.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/chevron_down.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/chevron_down_disabled.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/branch_open.svg</file>
+    <file>../icons/{{ if(cond=flavor.dark, t="dark", f="light") }}/branch_closed.svg</file>
```

Comment:
You might as well set a variable like `{% set icon_theme = if(cond=flavor.dark, t="dark", f="light") %}` and use it throughout. You could also use a for loop over an array of icons, though I'm not sure how often this list is being updated...